mkdocs.yml

```
@@ -97,9 +97,9 @@ nav:
   - Home: 
     - index.md
 
-  - Getting Started: 
+  - Get Started on NCI: 
```

```suggestion
  - Getting Started 
```
See a few comments above on the purpose of this section. 
Also, your suggested title would make the "Get Started on NCI" section title on the top bar (the blue one) much wider than the other sections' titles.